Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/main/home/TracingExplanationDialog.kt

```diff
@@ -0,0 +1,39 @@
+package de.rki.coronawarnapp.ui.main.home
+
+import android.content.Context
+import de.rki.coronawarnapp.R
+import de.rki.coronawarnapp.util.DialogHelper
+import javax.inject.Inject
+
+class TracingExplanationDialog @Inject constructor(
+    private val homeFragment: HomeFragment
+) {
+    private val context: Context
+        get() = homeFragment.requireContext()
+
+    fun show(activeTracingDaysInRetentionPeriod: Long, onPositive: () -> Unit) {
+        // get all text strings and the current active tracing time
+        val infoPeriodLogged =
+            context.getString(R.string.risk_details_information_body_period_logged)
+        val infoPeriodLoggedAssessment =
+            context.getString(
+                R.string.risk_details_information_body_period_logged_assessment,
+                activeTracingDaysInRetentionPeriod.toString()
+            )
+        val infoFAQ = context.getString(R.string.risk_details_explanation_dialog_faq_body)
+
+        // display the dialog
```

Comment:
I would prefer function calls over section comments

Reply:
Agree, neither seems necessary here though, I'll remove the comment.